LuaUI/widgets/chobby/core.lua

```diff
@@ -123,8 +123,7 @@ function Chobby:_PrintError(err)
 end
 
 function Chobby:_GetConfigData()
-	data = Configuration:SaveConfig()
-	return data
+	return Configuration:SaveConfig()
```

Comment:
also rename Configuratino's :SaveConfig to :GetConfigData. It does the same thing, just is named differently (might be confusing0